target/linux/ramips/image/mt7621.mk

```diff
@@ -220,6 +220,19 @@ define Device/buffalo_wsr-600dhp
 endef
 TARGET_DEVICES += buffalo_wsr-600dhp
 
+define Device/dlink_dir-8xx-a1
+  IMAGE_SIZE := 16000k
+  DEVICE_VENDOR := D-Link
+  DEVICE_VARIANT := A1
```

Comment:
I'd move the variant to the individual devices, as we don't know whether this will really be limited to A1, and I'd like to keep it with the DEVICE_MODEL variable.